src/applications/ivc-champva/10-7959f-1/config/form.js

```diff
@@ -94,11 +123,11 @@ const formConfig = {
             messageAriaDescribedby:
               'We use this information to verify other details.',
             veteranFullName: veteranFullNameUI,
-            veteranDateOfBirth: dateOfBirthUI(),
+            veteranDateOfBirth: dateOfBirthUI({ required: true }),
           },
           schema: {
             type: 'object',
-            required: ['fullName', 'veteranDOB'],
+            required: ['veteranFullName', 'veteranDateOfBirth'],
```

Comment:
Adjusted the `required` array so the keynames specified match the keynames in the uiSchema (did this in a couple other places as well)